src/pages/chat.tsx

```diff
@@ -0,0 +1,356 @@
+import type { NextPage } from 'next';
+import Image from 'next/image';
+import avatarMale from '../images/avatar/male.png';
+import avatarFemale from '../images/avatar/female.png';
+
+const Chat: NextPage = (props): JSX.Element => {
+    return (
+        <div>
+            <div className="h-screen w-screen bg-gradient-to-r from-complementary to-dominant flex justify-center items-center">
+                {/* <div className="flex-1 p:2 sm:p-6 justify-between flex flex-col h-4/5 w-6/12 bg-white"> */}
+                <div className="p:2 w-screen h-screen md:h-3/4 md:w-4/5 md:rounded-2xl lg:bg-white lg:rounded-2xl chat-lg justify-between flex flex-col bg-white">
+                    <div className="flex sm:items-center justify-between py-3 border-b-2 border-gray-200 pr-3">
+                        <div className="relative flex items-center space-x-4">
+                            <div className="relative pl-6 flex items-center">
+                                <div className="w-full h-full sm:w-16 sm:h-16 flex items-center">
+                                    <Image
+                                        src={avatarMale}
+                                        alt="Male Avatar"
+                                        className="avatar"
+                                    />
+                                </div>
+                            </div>
+                            <div className="flex flex-col leading-tight">
+                                <div className="text-2xl mt-1 flex items-center">
+                                    <span className="text-gray-700 mr-3">
+                                        StinkyGiraffe#7468
+                                    </span>
+                                </div>
+                            </div>
+                        </div>
+                        <div className="flex items-center space-x-2">
+                            {/* Menu Button */}
+                            <button
```

Comment:
What is this menu button? Will it have some specific behavior that someone has to implement?

Reply:
Once inside the chat page, since we don't have the usual navigation bar, I was hoping when that button is clicked, it would open up a list of different actions like signing out, resources page, etc.

Reply:
@piyasamyak @aasthabhatt00 Will this behavior be included in this PR as well?

Reply:
I think it's better to include it in a new PR, once this has been merged with main.

Reply:
Ok can you or Aastha make that ticket in JIRA?